src/main/resources/db/migration/V1_15__CASELOADS_DATA.sql

```diff
@@ -0,0 +1,165 @@
+Insert into CASELOADS (CASELOAD_ID,DESCRIPTION,CASELOAD_TYPE,LIST_SEQ,TRUST_ACCOUNTS_FLAG,ACCESS_PROPERTY_FLAG,TRUST_CASELOAD_ID,PAYROLL_FLAG,ACTIVE_FLAG,DEACTIVATION_DATE,COMMISSARY_FLAG,PAYROLL_TRUST_CASELOAD,COMMISSARY_TRUST_CASELOAD,TRUST_COMMISSARY_CASELOAD,COMMUNITY_TRUST_CASELOAD,MDT_FLAG,CREATE_DATETIME,CREATE_USER_ID,MODIFY_DATETIME,MODIFY_USER_ID,AUDIT_TIMESTAMP,AUDIT_USER_ID,AUDIT_MODULE_NAME,AUDIT_CLIENT_USER_ID,AUDIT_CLIENT_IP_ADDRESS,AUDIT_CLIENT_WORKSTATION_NAME,AUDIT_ADDITIONAL_INFO,CASELOAD_FUNCTION) values ('HPI','HIGHPOINT (HMP)','INST',138,'Y','Y',null,'N','Y',null,'N',null,null,null,null,'Y',to_timestamp('29-JUN-06 13.00.48.667206','DD-MON-RR HH24.MI.SSXFF6'),'OMS_OWNER',to_timestamp('30-JUL-16 18.13.50.599424','DD-MON-RR HH24.MI.SSXFF6'),'OMS_OWNER',to_timestamp('30-JUL-16 18.13.50.599469','DD-MON-RR HH24.MI.SSXFF6'),'OMS_OWNER','RD_V11.2.1.1.13','oracle','10.96.137.42',null,null,'GENERAL');
```

Comment:
taken directly from T3, we could just have subset of prisons but since there is less than 200 we have the whole lot